src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/organisations/OrganisationsSynchronisationService.kt

```diff
@@ -137,7 +137,24 @@ class OrganisationsSynchronisationService(
       }
     }
   }
-  suspend fun corporateAddressUpdated(event: CorporateAddressEvent) {}
+  suspend fun corporateAddressUpdated(event: CorporateAddressEvent) {
+    val telemetry = telemetryOf("nomisCorporateId" to event.corporateId, "dpsOrganisationId" to event.corporateId, "nomisAddressId" to event.addressId)
```

Comment:
`event.corporateId` used for both dps and nomis?

Reply:
Yes - but I looked as if they could be different